README.md

```diff
@@ -78,12 +78,17 @@ If you have a variable in your system environment that you want injected in an a
 
 ## Installation
 
+The correct version for your needs depends on the version of Distillery your project uses:
+
+- For Distillery versions `2.1.0` and later, you want `{:aws_ssm_provider, "~> 0.3.0"}`.
+- For Distillery versions `2.0.14` and earlier, you want `{:aws_ssm_provider, "~> 0.2.1"}`.
+
 The package can be installed by adding `aws_ssm_provider` to your list of dependencies in `mix.exs`:
 
 ```elixir
 def deps do
   [
-    {:aws_ssm_provider, "~> 0.1.3"}
+    {:aws_ssm_provider, "~> 0.3.0"}
```

Comment:
breaking changes require a major release bump https://semver.org/

Reply:
Oh. You mean MAJOR. Elixir is still on major version 1. Okay, I'll do that.

Reply:
Updated to 1.0.0